fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/autobrakes.rs

```diff
@@ -504,21 +516,72 @@ impl A380AutobrakeController {
             lgciu2,
         );
 
-        self.mode = self.determine_mode(context, autobrake_panel);
+        self.braking_distance_calculator.update_braking_estimations(
+            context,
+            self.ground_speed,
+            if self.mode == A380AutobrakeMode::BTV {
+                self.btv_scheduler.predicted_decel()
+            } else if self.mode != A380AutobrakeMode::DISARM {
+                context.long_accel()
+            } else {
+                Acceleration::default()
+            },
```

Comment:
Could be a ``match`` here? 

```
match self.mode {
    A380AutobrakeMode::BTV => self.btv_scheduler.predicted_decel(),
    A380AutobrakeMode::DISARM => Acceleration::default(),
    _ => context.long_accel(),
},
```
```suggestion
            if self.mode == A380AutobrakeMode::BTV {
                self.btv_scheduler.predicted_decel()
            } else if self.mode != A380AutobrakeMode::DISARM {
                context.long_accel()
            } else {
                Acceleration::default()
            },
```

Reply:
This breaks the coding guideline: https://github.com/flybywiresim/aircraft/blob/master/fbw-common/src/wasm/systems/guidelines.md#explicit-match-pattern-declaration

Reply:
Then the equivalent using ``match`` but having ``_`` => lead to a ``panic!`` would meet this guideline correct?

Reply:
This would meet the guideline yes. But then one can just stay with the if statement as well.